pkg/kube/services/services.go

```diff
@@ -92,6 +96,49 @@ func GetServiceURLFromMap(services map[string]*v1.Service, name string) string {
 	return GetServiceURL(services[name])
 }
 
+func GetIstioVirtualService(namespace string, name string) (*unstructured.Unstructured, error) {
```

Comment:
replace with namespace, name string -> gocritic will flag this in the linter.

Reply:
Does this function need to be exported from this package? Seems like it's only used for `GetUrlFromVirtualService` which is in the same package ...

Reply:
yupp we can do that